src/platform/qt/Window.cpp

```diff
@@ -592,7 +602,7 @@ void Window::consoleOpen() {
 #ifdef ENABLE_SCRIPTING
 void Window::scriptingOpen() {
 	ensureScripting();
-	ScriptingView* view = new ScriptingView(m_scripting.get(), m_config);
+	ScriptingView* view = new ScriptingView(d->scripting.get(), m_config);
```

Comment:
It's worth observing that I did _not_ deduplicate the scripting view. It wouldn't have been difficult to do so but it's one that someone might reasonably WANT two of.